netlify.toml

```diff
@@ -0,0 +1,22 @@
+# BUILD Settings:
+# ##############################################################################
+[build]
+  publish = "styleguide/out"
+  functions = "lambda"
+
+[dev]
+  publish = "styleguide/out"
+  port = 64946
+  autolaunch = true
+
+# HEADERS - BASIC AUTH
+# ##############################################################################
+[[headers]]
+  for = "/*"
+  [headers.values]
+    Access-Control-Allow-Origin = "*"
+    Content-Security-Policy = "form-action https:"
+    X-Content-Type-Options = "nosniff"
+    Referrer-Policy = "origin-when-cross-origin"
+    Strict-Transport-Security = "max-age=2592000"
+    Permissions-Policy = "vibrate=(), geolocation=(), midi=(), notifications=(), push=(), microphone=(), camera=(), magnetometer=(), gyroscope=(), speaker=(), vibrate=(), fullscreen=()"
```

Comment:
I'm wondering if we could enable fullscreen for testing those youtube embeds? Actually doesn't matter as much here in the Decanter repo, but I wouldn't mind having it back for the Giving repo for that purpose 🤔

Reply:
Ah yes. Probably can open a few of these up. These were just blanket turn off all the things.